src/imap.rs

```diff
@@ -671,6 +671,11 @@ impl Imap {
         folder: &str,
         fetch_existing_msgs: bool,
     ) -> Result<bool> {
+        if should_ignore_folder(context, folder).await? {
+            info!(context, "Not fetching from {}", folder);
+            return Ok(false);
+        }
```

Comment:
Is this really sufficient?  shouldn't we also be idling on the mvbox as well if this option is true.  otherwise they'll receive messages sporadically only I think?

Reply:
Yes it's sufficient because if the user sets only_fetch_mvbox, mvbox_move we enabled automatically (which is what we've been discussing a lot), which in turn means that we will idle on the mvbox.